CMakeLists.txt

```diff
@@ -394,8 +406,11 @@ option(WITH_RADOSGW_AMQP_ENDPOINT "Rados Gateway's pubsub support for AMQP push
 option(WITH_RADOSGW_KAFKA_ENDPOINT "Rados Gateway's pubsub support for Kafka push endpoint" ON)
 option(WITH_RADOSGW_LUA_PACKAGES "Rados Gateway's support for dynamically adding lua packagess" ON)
 option(WITH_RADOSGW_DBSTORE "DBStore backend for Rados Gateway" ON)
-option(WITH_RADOSGW_MOTR "CORTX-Motr backend for Rados Gateway" ON)
-option(WITH_RADOSGW_SELECT_PARQUET "Support for s3 select on parquet objects" OFF)
+option(WITH_RADOSGW_MOTR "CORTX-Motr backend for Rados Gateway" OFF)
```

Comment:
@shailesh-vaidya, FYI - it will not build with Motr by default any more.

Reply:
In the confluence page which I created it's already mentioned that `-DWITH_RADOSGW_MOTR=YES` should be used. I don't know about other pages or scripts. But this change was requested by @shailesh-vaidya. I suppose, he is responsible for the build scripts, no? If not or not only him - please let others know. Thanks.